[0m[[0m[0mdebug[0m] [0m[0m> Exec(run, Some(6732730e-c923-4b3b-951d-c307d39f7d0a), Some(CommandSource(console0)))[0m
[0m[[0m[0mdebug[0m] [0m[0mEvaluating tasks: Compile / run[0m
[0m[[0m[0mdebug[0m] [0m[0mRunning task... Cancel: Signal, check cycles: false, forcegc: true[0m
[0m[[0m[31merror[0m] [0m[0morg.apache.flink.runtime.client.JobExecutionException: Job execution failed.[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.runtime.jobmaster.JobResult.toJobExecutionResult(JobResult.java:144)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.runtime.minicluster.MiniClusterJobClient.lambda$getJobExecutionResult$3(MiniClusterJobClient.java:141)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:642)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2073)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.runtime.rpc.akka.AkkaInvocationHandler.lambda$invokeRpc$1(AkkaInvocationHandler.java:268)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:859)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:837)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2073)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.util.concurrent.FutureUtils.doForward(FutureUtils.java:1277)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.runtime.concurrent.akka.ClassLoadingUtils.lambda$null$1(ClassLoadingUtils.java:93)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.runtime.concurrent.akka.ClassLoadingUtils.runWithContextClassLoader(ClassLoadingUtils.java:68)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.runtime.concurrent.akka.ClassLoadingUtils.lambda$guardCompletionWithContextClassLoader$2(ClassLoadingUtils.java:92)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:859)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:837)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2073)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.runtime.concurrent.akka.AkkaFutureUtils$1.onComplete(AkkaFutureUtils.java:47)[0m
[0m[[0m[31merror[0m] [0m[0m	at akka.dispatch.OnComplete.internal(Future.scala:300)[0m
[0m[[0m[31merror[0m] [0m[0m	at akka.dispatch.OnComplete.internal(Future.scala:297)[0m
[0m[[0m[31merror[0m] [0m[0m	at akka.dispatch.japi$CallbackBridge.apply(Future.scala:224)[0m
[0m[[0m[31merror[0m] [0m[0m	at akka.dispatch.japi$CallbackBridge.apply(Future.scala:221)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:60)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.runtime.concurrent.akka.AkkaFutureUtils$DirectExecutionContext.execute(AkkaFutureUtils.java:65)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:68)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1(Promise.scala:284)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.concurrent.impl.Promise$DefaultPromise.$anonfun$tryComplete$1$adapted(Promise.scala:284)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:284)[0m
[0m[[0m[31merror[0m] [0m[0m	at akka.pattern.PromiseActorRef.$bang(AskSupport.scala:621)[0m
[0m[[0m[31merror[0m] [0m[0m	at akka.pattern.PipeToSupport$PipeableFuture$$anonfun$pipeTo$1.applyOrElse(PipeToSupport.scala:24)[0m
[0m[[0m[31merror[0m] [0m[0m	at akka.pattern.PipeToSupport$PipeableFuture$$anonfun$pipeTo$1.applyOrElse(PipeToSupport.scala:23)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.concurrent.Future.$anonfun$andThen$1(Future.scala:532)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:29)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:29)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:60)[0m
[0m[[0m[31merror[0m] [0m[0m	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:63)[0m
[0m[[0m[31merror[0m] [0m[0m	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:100)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:81)[0m
[0m[[0m[31merror[0m] [0m[0m	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:100)[0m
[0m[[0m[31merror[0m] [0m[0m	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:49)[0m
[0m[[0m[31merror[0m] [0m[0m	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)[0m
[0m[[0m[31merror[0m] [0m[0mCaused by: org.apache.flink.runtime.JobException: Recovery is suppressed by NoRestartBackoffTimeStrategy[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.runtime.executiongraph.failover.flip1.ExecutionFailureHandler.handleFailure(ExecutionFailureHandler.java:139)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.runtime.executiongraph.failover.flip1.ExecutionFailureHandler.getGlobalFailureHandlingResult(ExecutionFailureHandler.java:102)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.runtime.scheduler.DefaultScheduler.handleGlobalFailure(DefaultScheduler.java:299)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.runtime.operators.coordination.OperatorCoordinatorHolder$LazyInitializedCoordinatorContext.lambda$failJob$0(OperatorCoordinatorHolder.java:635)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.runtime.rpc.akka.AkkaRpcActor.lambda$handleRunAsync$4(AkkaRpcActor.java:453)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.runtime.concurrent.akka.ClassLoadingUtils.runWithContextClassLoader(ClassLoadingUtils.java:68)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.runtime.rpc.akka.AkkaRpcActor.handleRunAsync(AkkaRpcActor.java:453)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.runtime.rpc.akka.AkkaRpcActor.handleRpcMessage(AkkaRpcActor.java:218)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.runtime.rpc.akka.FencedAkkaRpcActor.handleRpcMessage(FencedAkkaRpcActor.java:84)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.runtime.rpc.akka.AkkaRpcActor.handleMessage(AkkaRpcActor.java:168)[0m
[0m[[0m[31merror[0m] [0m[0m	at akka.japi.pf.UnitCaseStatement.apply(CaseStatements.scala:24)[0m
[0m[[0m[31merror[0m] [0m[0m	at akka.japi.pf.UnitCaseStatement.apply(CaseStatements.scala:20)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.PartialFunction.applyOrElse(PartialFunction.scala:123)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.PartialFunction.applyOrElse$(PartialFunction.scala:122)[0m
[0m[[0m[31merror[0m] [0m[0m	at akka.japi.pf.UnitCaseStatement.applyOrElse(CaseStatements.scala:20)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.PartialFunction$OrElse.applyOrElse(PartialFunction.scala:171)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.PartialFunction$OrElse.applyOrElse(PartialFunction.scala:172)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.PartialFunction$OrElse.applyOrElse(PartialFunction.scala:172)[0m
[0m[[0m[31merror[0m] [0m[0m	at akka.actor.Actor.aroundReceive(Actor.scala:537)[0m
[0m[[0m[31merror[0m] [0m[0m	at akka.actor.Actor.aroundReceive$(Actor.scala:535)[0m
[0m[[0m[31merror[0m] [0m[0m	at akka.actor.AbstractActor.aroundReceive(AbstractActor.scala:220)[0m
[0m[[0m[31merror[0m] [0m[0m	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:580)[0m
[0m[[0m[31merror[0m] [0m[0m	at akka.actor.ActorCell.invoke(ActorCell.scala:548)[0m
[0m[[0m[31merror[0m] [0m[0m	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:270)[0m
[0m[[0m[31merror[0m] [0m[0m	at akka.dispatch.Mailbox.run(Mailbox.scala:231)[0m
[0m[[0m[31merror[0m] [0m[0m	at akka.dispatch.Mailbox.exec(Mailbox.scala:243)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)[0m
[0m[[0m[31merror[0m] [0m[0mCaused by: org.apache.flink.util.FlinkException: Global failure triggered by OperatorCoordinator for 'Source: Kafka Source -> Sink: Print to Std. Out' (operator cbc357ccb763df2852fee8c4fc7d55f2).[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.runtime.operators.coordination.OperatorCoordinatorHolder$LazyInitializedCoordinatorContext.failJob(OperatorCoordinatorHolder.java:617)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.runtime.operators.coordination.RecreateOnResetOperatorCoordinator$QuiesceableContext.failJob(RecreateOnResetOperatorCoordinator.java:237)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.runtime.source.coordinator.SourceCoordinatorContext.failJob(SourceCoordinatorContext.java:360)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.runtime.source.coordinator.SourceCoordinatorContext.handleUncaughtExceptionFromAsyncCall(SourceCoordinatorContext.java:373)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.util.ThrowableCatchingRunnable.run(ThrowableCatchingRunnable.java:42)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.lang.Thread.run(Thread.java:834)[0m
[0m[[0m[31merror[0m] [0m[0mCaused by: org.apache.flink.util.FlinkRuntimeException: Failed to list subscribed topic partitions due to [0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.connector.kafka.source.enumerator.KafkaSourceEnumerator.checkPartitionChanges(KafkaSourceEnumerator.java:234)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.runtime.source.coordinator.ExecutorNotifier.lambda$null$1(ExecutorNotifier.java:83)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.util.ThrowableCatchingRunnable.run(ThrowableCatchingRunnable.java:40)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.lang.Thread.run(Thread.java:834)[0m
[0m[[0m[31merror[0m] [0m[0mCaused by: java.lang.RuntimeException: Failed to get metadata for topics [flink-example].[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.connector.kafka.source.enumerator.subscriber.KafkaSubscriberUtils.getTopicMetadata(KafkaSubscriberUtils.java:47)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.connector.kafka.source.enumerator.subscriber.TopicListSubscriber.getSubscribedTopicPartitions(TopicListSubscriber.java:52)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.connector.kafka.source.enumerator.KafkaSourceEnumerator.getSubscribedTopicPartitions(KafkaSourceEnumerator.java:219)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.runtime.source.coordinator.ExecutorNotifier.lambda$notifyReadyAsync$2(ExecutorNotifier.java:80)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.lang.Thread.run(Thread.java:834)[0m
[0m[[0m[31merror[0m] [0m[0mCaused by: java.util.concurrent.ExecutionException: org.apache.kafka.common.errors.TimeoutException: Timed out waiting for a node assignment. Call: describeTopics[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:395)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:1999)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.kafka.common.internals.KafkaFutureImpl.get(KafkaFutureImpl.java:165)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.connector.kafka.source.enumerator.subscriber.KafkaSubscriberUtils.getTopicMetadata(KafkaSubscriberUtils.java:44)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.connector.kafka.source.enumerator.subscriber.TopicListSubscriber.getSubscribedTopicPartitions(TopicListSubscriber.java:52)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.connector.kafka.source.enumerator.KafkaSourceEnumerator.getSubscribedTopicPartitions(KafkaSourceEnumerator.java:219)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.flink.runtime.source.coordinator.ExecutorNotifier.lambda$notifyReadyAsync$2(ExecutorNotifier.java:80)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.lang.Thread.run(Thread.java:834)[0m
[0m[[0m[31merror[0m] [0m[0mCaused by: org.apache.kafka.common.errors.TimeoutException: Timed out waiting for a node assignment. Call: describeTopics[0m
[0m[[0m[31merror[0m] [0m[0m(Compile / [31mrun[0m) org.apache.flink.runtime.client.JobExecutionException: Job execution failed.[0m
[0m[[0m[31merror[0m] [0m[0mTotal time: 71 s (01:11), completed 23-Jul-2023, 4:40:40 PM[0m
[0m[[0m[0mdebug[0m] [0m[0m> Exec(shell, None, None)[0m
[0m[[0m[0mdebug[0m] [0m[0mForcing garbage collection...[0m
